tests/barline_tests.ts

```diff
@@ -33,7 +28,7 @@ function simple(options: TestOptions): void {
 
   const notes = [
     f.StaveNote({ keys: ['d/4', 'e/4', 'f/4'], stem_direction: -1, duration: '2' }),
-    f.BarNote({ type: 'single' }), // => f.BarNote({ type: BarlineType.SINGLE })
+    f.BarNote({ type: 'single' }),
```

Comment:
Oh, my comment here was a question: should we change this line of code to use the BarlineType enum, for enhanced compiler checks? Or do we prefer developers to stick with 'single' (it's shorter to type, for sure!).

Reply:
I saw that `BarNote` supported internally in `SetType` both possibilities and I adapted the parameter of the constructor.